### PR TITLE
cmd/internal: elide issue assignee based on authorship

### PR DESCRIPTION
**Pull request by irfansharif**
Touches #51653. We've been spamming Andrei because of his refactor of everything related to roach{test/prod}, which makes it difficult to disambiguate which issues should actually be assigned to him vs. not. We replace roachtest assignment with new labels instead that hopefully are a bit more useful going forward. Unit test flakes still default to using `git blame` style ownership, but that's a yak we can shave elsewhere.

Release note: None

### CODE REVIEW

**Review by jordanlewis — CHANGES_REQUESTED** [review]
Thumbs down on this because AFAIK it will also affect non-roachtest failures, and I don't think we've decided that those are bad yet as well.

Would you be able to only elide the issue assignment if you detect that it's `roachtest` itself failing?

**Review by irfansharif — COMMENTED** [review]
Wanna take another look? Should be just roachtests now.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/51660#-:-MCn-C7PBiffrcpa0Uo7:bwrar8k)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @tbg)

File: pkg/cmd/roachtest/test_registry.go [hunk 1]
```diff
@@ -44,19 +44,50 @@ const (
...
-	OwnerAppDev:       {SlackRoom: `app-dev`, ContactEmail: `rafi@cockroachlabs.com`},
-	OwnerBulkIO:       {SlackRoom: `bulk-io`, ContactEmail: `david@cockroachlabs.com`},
-	OwnerCDC:          {SlackRoom: `cdc`, ContactEmail: `ajwerner@cockroachlabs.com`},
-	OwnerKV:           {SlackRoom: `kv`, ContactEmail: `andrei@cockroachlabs.com`},
-	OwnerPartitioning: {SlackRoom: `partitioning`, ContactEmail: `andrei@cockroachlabs.com`},
-	OwnerSQLExec:      {SlackRoom: `sql-execution-team`, ContactEmail: `alfonso@cockroachlabs.com`},
-	OwnerSQLSchema:    {SlackRoom: `sql-schema`, ContactEmail: `lucy@cockroachlabs.com`},
-	OwnerStorage:      {SlackRoom: `storage`, ContactEmail: `peter@cockroachlabs.com`},
+	OwnerAppDev: {SlackRoom: `app-dev`, ContactEmail: `rafi@cockroachlabs.com`,
+		TriageColumnID: 8532151,
```
Comment (irfansharif) [inline pkg/cmd/roachtest/test_registry.go:67]: Nice jankiness. Maybe just add a comment next to each one naming the specific board/column.

**Review by irfansharif — COMMENTED** [review]
Reviewing my own PR here but LGTM, let's get another pair of eyes on it given @tbg is out for a few days and get it in?

**Review by jordanlewis — APPROVED** [review]
LGTM :trollface:

**Pull request merged**